.gitignore

```diff
@@ -99,3 +99,5 @@ __pycache__/
 /rdkit/ML/Data/test_data/testgeneral.dat.pkl
 /rdkit/ML/Data/test_data/testquant.qdat.pkl
 .DS_Store
+.idea
```

Comment:
Please don't add files to the hosted `.gitignore` that are specific to your system.
These changes aren't related to the PR you're making

Reply:
Note to self: don't use StackOverflow solutions without thinking about them first! Of course, .gitignore is as much a part of the repository as anything else. I'll look through my IDE settings and see if there's a way to not include the .idea files from there instead.